updatecli/updatecli.d/terraform-providers/vpc.yaml

```diff
@@ -0,0 +1,50 @@
+name: Bump version of the Terraform "vpc" module in terraform-aws-sponsorship/vpc.tf
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  getLatestVersion:
+    kind: githubrelease
```

Comment:
```suggestion
    kind: terraform/registry
```

Need addition adaptation in the `spec` section of course. Sources of informations to help you:
- #25 
- https://www.updatecli.io/docs/plugins/resource/terraform/registry/ (attribute `type` )